lib/rails_cloud_tasks/configuration.rb

```diff
@@ -20,7 +22,7 @@ def inject_routes
     private
 
     def authenticate
-      email = AppEngine.service_account_email || Google::Auth.get_application_default.issuer
+      email = @service_account_email || Google::Auth.get_application_default.issuer
```

Comment:
This is an attr_accessor, you don't need the @

Reply:
This method has a recue clause on `Errno::EHOSTDOWN`. It should be moved to the initializer, since you also moved the AppEngine call.